apps/cookbook/src/app/showcase/loading-overlay-showcase/loading-overlay-showcase.component.ts

```diff
@@ -1,8 +1,6 @@
 import { Component } from '@angular/core';
-import { ApiDescriptionProperty } from '~/app/shared/api-description/api-description-properties/api-description-properties.component';
-
 import exampleHtml from '../../examples/loading-overlay-example/loading-overlay-example.component.html?raw';
-
+import { ApiDescriptionProperty } from '../../shared/api-description/api-description-properties/api-description-properties.component';
 @Component({
```

Comment:
So minor, but there is a missing space between the component decorator and the import. Should prettier not take care of that?

Reply:
I can't find any Prettier config options to take care of that 🤔